scripts/emr/terraform/variables.tf.json

```diff
@@ -0,0 +1,33 @@
+{ 
+  "variable": {
+    "access_key": { 
+        "description": "From your `~/.aws/credentials"
+    },
```

Comment:
Why are the access key and secret key needed?  Shouldn't those be picked up from the environment?

Reply:
Look into two last @jpolchlo commits, he already introduced this feature. I prefer to configure env using configuration files though.

Reply:
Looking from a strictly behavioral point of view, all I know is that I was prompted for those things when I tried to run `terraform apply`.  Providing them through environment variables is possible (I did it), but why are they needed at all?  The AWS credentials can be picked up from the environment (please see the terraform configuration [here](https://github.com/geodocker/geodocker-jupyter-geopyspark)).

Reply:
@jamesmcclain sry your link is broken

Reply:
Btw I don't use global variables, would it pick up my `.aws/credentials`?

Reply:
I was able to click on it successfully.  https://github.com/geodocker/geodocker-jupyter-geopyspark

Reply:
Ah i thought it was a link on their docs.

Reply:
I am referring in particular to [this](https://github.com/geodocker/geodocker-jupyter-geopyspark/tree/799bc76957c8370d728d6344ed5d8db940744c92/emr-nodocker/terraform) and [this](https://github.com/geodocker/geodocker-jupyter-geopyspark/tree/799bc76957c8370d728d6344ed5d8db940744c92/terraform).  (Be advised that I have a PR up right now will change the directory structure.)